src/Utilities/Workspaces/CodeMetrics/CodeAnalysisMetricDataExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.CodeMetrics
+{
+    internal static class CodeAnalysisMetricDataExtensions
```

Comment:
The name suggests it contains extension methods.

Reply:
I created the class before realizing the method I was moving was static 😊

Reply:
I have a solution though...

Reply:
Fixed